src/pages/LoginPage.vue

```diff
@@ -50,11 +59,23 @@ export default defineComponent({
           router.push('/');
         })
         .catch((error) => {
-          console.log('Error', error.response);
+          console.log(error);
+
+          if (error.response.data.status === 'failed') {
+            if (error.response.data.errorCode === 'FAILED_TO_FETCH_USER') {
+              errorMessage.value = 'Wrong Password or Username'; // Update error message
+            } else if (error.response.data.errorCode === 'BAD_CREDENTIALS') {
+              errorMessage.value = 'Wrong Password or Username'; // Update error message
+            } else if (error.response.data.errorCode === 'NO_USER') {
+              errorMessage.value = 'Wrong Password or Username'; // Update error message
```

Comment:
Jeg tænker at man her godt kan sige at useren ikke findes så: "No User Exists With This Username"